Validation/Configuration/python/postValidation_cff.py

```diff
@@ -73,15 +73,19 @@
     electronPostValidationSequenceMiniAOD
 )
 
-def _modifyPostValidationForPhase2( theProcess ):
-    theProcess.load('Validation.MuonGEMHits.PostProcessor_cff')
-    theProcess.load('Validation.MuonGEMDigis.PostProcessor_cff')
-    theProcess.load('Validation.MuonGEMRecHits.PostProcessor_cff')
-    theProcess.load('Validation.HGCalValidation.HGCalPostProcessor_cff')
-    theProcess.postValidation += theProcess.MuonGEMHitsPostProcessors
-    theProcess.postValidation += theProcess.MuonGEMDigisPostProcessors
-    theProcess.postValidation += theProcess.MuonGEMRecHitsPostProcessors
-    theProcess.postValidation += theProcess.hgcalPostProcessor
+from Validation.MuonGEMHits.PostProcessor_cff import *
+from Validation.MuonGEMDigis.PostProcessor_cff import *
+from Validation.MuonGEMRecHits.PostProcessor_cff import *
+from Validation.HGCalValidation.HGCalPostProcessor_cff import *
+
+_run3_postValidation = postValidation.copy()
+_run3_postValidation += MuonGEMHitsPostProcessors
+_run3_postValidation += MuonGEMDigisPostProcessors
+_run3_postValidation += MuonGEMRecHitsPostProcessors
+
+_phase2_postValidation = _run3_postValidation.copy()
+_phase2_postValidation += hgcalPostProcessor
 
 from Configuration.StandardSequences.Eras import eras
-modifyConfigurationStandardSequencesPostValidationForPhase2_ = eras.phase2_muon.makeProcessModifier( _modifyPostValidationForPhase2 )
+eras.run3_GEM.toReplaceWith( postValidation, _run3_postValidation )
```

Comment:
@kpedro88 - also for the future - none of this should be on the post validation sequence - as nothing here depends on hlt - it can be on the prevalidation sequence.